manifests/cli/params.pp

```diff
@@ -12,13 +12,11 @@
   # OS specific paths
   case $::osfamily {
     'RedHat': {
-      case $::operatingsystem {
-        'fedora': {
-          $hammer_plugin_prefix = 'rubygem-hammer_cli_'
-        }
-        default: {
-          $hammer_plugin_prefix = 'tfm-rubygem-hammer_cli_'
-        }
+      # We use system packages except on EL7
+      if versioncmp($facts['operatingsystemmajrelease'], '8') >= 0 {
```

Comment:
When you say "except on EL7", maybe we should check for EL7?
```suggestion
      if $facts['operatingsystemmajrelease'] == '7' {
```

Reply:
(and obviously flip that if/else around)

Reply:
I preferred this version because it's a smaller diff if you ignore whitespace, but did think about your suggestion before. I was also worried about strings vs integers in an exact comparison.